src/compiler/config/test/validate-config.spec.ts

```diff
@@ -256,10 +257,13 @@ describe('validation', () => {
     expect(config.devMode).toBe(false);
   });
 
-  it('should set default generateDocs to false', () => {
-    const { config } = validateConfig(userConfig);
-    expect(config.outputTargets.some((o) => o.type === 'docs')).toBe(false);
-  });
+  it.each([DOCS_JSON, DOCS_CUSTOM, DOCS_README, DOCS_VSCODE])(
+    'should not add "%s" output target by default',
+    (targetType) => {
+      const { config } = validateConfig(userConfig);
+      expect(config.outputTargets.some((o) => o.type === targetType)).toBe(false);
+    }
+  );
```

Comment:
there were two identical tests both testing what I believe to be an outdated / deprecated output target (`"docs"`), I replaced them with this one checking that none of the doc-related output targets get automatically added